### PR TITLE
feat: schedule benchmark and sync-mainnet

### PR DESCRIPTION
**Pull request by keroro520**
Report the results of each day's "Benchmark" and "Sync-Mainnet" to https://github.com/nervosnetwork/ckb/pull/2372

### CODE REVIEW

File: .github/workflows/benchmark.yml [hunk 1]
```diff
@@ -4,15 +4,19 @@ on:
...
   pull_request_review:
     types: [ submitted ]
+  schedule:
+    - cron: '0 0 * * *'
 
 jobs:
   build_modified_ckb:
     name: Build Modified CKB
-    runs-on: [ self-hosted, Linux, X64, building ]
+    runs-on: ubuntu-latest
```
Comment (keroro520) [inline .github/workflows/benchmark.yml:13]: I change the runner from self-hosted to github-hosted. The reason for this is that "Benchmark" and "Sync-Mainnet" are low-frequency tests. We can run multiple tests at the same time using github-hosted runners.

**Review by keroro520 — COMMENTED** [review]
#benchmark

**Review by keroro520 — COMMENTED** [review]
#benchmark

**Review by liya2017 — APPROVED** [review]
LGTM

File: .github/workflows/sync.yml [hunk 2]
```diff
@@ -3,15 +3,22 @@ name: Sync-Mainnet
...
-    runs-on: [ self-hosted, Linux, X64, building ]
+    runs-on: ubuntu-latest
     if: |
-      github.repository_owner == 'nervosnetwork' &&
-      contains(github.event.review.body, '#sync-mainnet') &&
-      contains(fromJson( '[ "janx", "doitian", "quake", "xxuejie", "zhangsoledad", "jjyr", "TheWaWaR", "driftluo", "keroro520", "yangby-cryptape", "liya2017" ]' ), github.actor)
+      (
+        github.event_name == 'pull_request_review' &&
+        contains(github.event.review.body, '#sync-mainnet') &&
+        contains(fromJson('[ "janx", "doitian", "quake", "xxuejie", "zhangsoledad", "jjyr", "TheWaWaR", "driftluo", "keroro520", "yangby-cryptape", "liya2017" ]'), github.actor)
```
Comment (yangby-cryptape) [inline .github/workflows/sync.yml:17]: I find a way to remove this ugly code, but a little complex, I will pull request after your PR merged. 😄

**Comment by keroro520** [comment]
@doitian Please take a look.

**Review by doitian — APPROVED** [review]

**Comment by doitian** [comment]
bors r=doitian,liya2017

**Pull request merged**